system/load_average.tf

```diff
@@ -0,0 +1,71 @@
+resource "datadog_monitor" "load_average_1" {
+  count   = "${var.monitor_enabled}"
+  name    = "High load average 1 ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High load average 1 last ${var.load_average["time"]} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.public_ip}"
+  query   = "avg(last_${var.load_average["time"]}):avg:system.load.1{host:${data.aws_instance.monitored.instance_id}} by {host} > ${var.load_average["critical_threshold"]}"
+
+  thresholds {
+    ok       = "${var.load_average["ok_threshold"]}"
+    warning  = "${var.load_average["warning_threshold"]}"
+    critical = "${var.load_average["critical_threshold"]}"
+  }
+
+  renotify_interval = "${var.renotify_interval_mins}"
+  new_host_delay    = "${var.new_host_delay}"
+  notify_no_data    = "${var.notify_no_data}"
+
+  silenced {
+    "*" = "${var.monitor_silenced}"
+  }
+
+  tags = ["${module.label.id}"]
```

Comment:
`.tags` not `.id`

Reply:
`tags =` can't be map

Reply:
Ah, right, these are `datadog` tags. In that case, it should be in this format:

![image](https://user-images.githubusercontent.com/52489/30953885-b7b4bae6-a3e2-11e7-843f-3d31ff7907f8.png)

Reply:
`key:value`